package.json

```diff
@@ -1,52 +1,43 @@
 {
-  "name": "cpr-multiselect",
-  "version": "3.2.2",
-  "description": "React component for selecting multiple entities from a drop-down",
-  "main": "build/multi-selector.js",
-  "scripts": {
-    "build": "webpack",
-    "prepublish": "npm run build",
-    "watch": "webpack --watch"
-  },
-  "sofe": {
-    "authService": "http://canopytax.com/someservice.js"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/CanopyTax/cpr-multiselect.git"
-  },
-  "author": "Bret Little",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/CanopyTax/cpr-multiselect/issues"
-  },
-  "homepage": "https://github.com/CanopyTax/cpr-multiselect",
-  "devDependencies": {
-    "autoprefixer-loader": "1.2.0",
-    "babel": "5.8.23",
-    "babel-core": "5.8.25",
-    "babel-loader": "5.3.2",
-    "css-loader": "^0.20.2",
-    "jasmine-core": "2.3.4",
```

Comment:
the testing dependencies were probably copy/pasted from another repo. There aren't any tests defined within this repo, so I removed the test-related dev dependencies